mapflow/mapflow.py

```diff
@@ -1014,6 +1031,7 @@ def request_mapflow_metadata(self,
 
     def request_mapflow_metadata_error_handler(self, response: QNetworkReply):
         self.report_http_error(response,
+                               response.readAll().data().decode(),
```

Comment:
There is going to be a lot simmilar shanges. 
A long time ago I added an argument to 'report_http_error', but did not change it in all occurrences. 
Noticed this only now - and fixed it.

Reply:
I remember I had a problem with reading it inside 'report_http_error' and not passing it there as an argument. But now I tried again and it works...
So I'll change it back